app/install.php

```diff
@@ -362,9 +362,24 @@ function printStep0() {
 <?php
 }
 
+function getProcessUsername() {
+	if (function_exists('posix_getpwuid') && function_exists('posix_geteuid')) {
+		$processUser = posix_getpwuid(posix_geteuid());
+		return $processUser['name'];
+	}
+
```

Comment:
```suggestion

$output = null;
```

Reply:
I think there is no need for that with the test I've added to check if the function exists or not